### ISSUE TITLE
Link to documentation broken

### ISSUE THREAD

**Issue by arbenede** [FluxML/DiffImages.jl#28]
The link [https://somtambe.github.io/DiffImages.jl/dev/](https://somtambe.github.io/DiffImages.jl/dev/) is broken, could you please fix it?

**Comment by SomTambe** [comment]
I agree. The link should be changed to [https://fluxml.ai/DiffImages.jl/dev/](https://fluxml.ai/DiffImages.jl/dev/).

Can you rectify this? @DhairyaLGandhi

**Comment by roflmaostc** [comment]
bump
